integration/docker/entrypoint.sh

```diff
@@ -129,7 +129,7 @@ case ${service,,} in
   fuse)
     # Unmount first if cleanup failed and ignore error
     ! integration/fuse/bin/alluxio-fuse unmount /alluxio-fuse
-    integration/fuse/bin/alluxio-fuse mount -o allow_other /alluxio-fuse /
+    integration/fuse/bin/alluxio-fuse mount ${options} /alluxio-fuse /
```

Comment:
Agreed that we can allow passing mount point options through docker entrypoint, but using existing `${options}` seems a little confusing. For all other commands, options refer to `--no-format`, but for fuse, it refers to fuse mount options. Maybe we  can pass in `fuse --fuse-opts=<fuse options separated by comma>`? Could you also update the usage docs?

Reply:
will do

Reply:
done

and as i change the entrypoint.sh so the image of alluxio-fuse have to release too ...